lib/broken_link_report.rb

```diff
@@ -0,0 +1,63 @@
+class BrokenLinkReport
+  attr_reader :flows_root_path
+  def initialize(flows_root_path)
+    @flows_root_path = flows_root_path
+  end
+
+  def folder_paths
+    path_to_items_within_root_folder = File.join(flows_root_path, "*")
+    Dir.glob(path_to_items_within_root_folder).select { |file| File.directory?(file) }
```

Comment:
Does the flows_root_path ever change? Could we simplify this to -

```suggestion
  def folder_paths
    flows_path = Rails.root.join("lib/smart_answer_flows/*")
    Dir.glob(flows_path).select { |file| File.directory?(file) }
```

Reply:
It makes it harder to test. In test the flow root is at "test/test/fixtures/smart_answer_flows" and "test/fixtures/smart_answer_flows/flow-with-links" thereby testing against folders with limited content and with a relatively fixed content.

Reply:
It also makes it easier to extend the behaviour (for example, if want a report of just one flow you can set the root as the flow folder).